spec/avram/migrator/runner_spec.cr

```diff
@@ -11,4 +11,37 @@ describe Avram::Migrator::Runner do
       end
     end
   end
+
+  describe ".create_db" do
+    context "when the DB doesn't exist yet" do
+      it "creates the new DB" do
+        Avram.temp_config(database_to_migrate: SampleBackupDatabase) do
+          Avram::Migrator::Runner.create_db(quiet?: true)
+
+          DB.open(SampleBackupDatabase.credentials.url) do |db|
```

Comment:
This fully closes the DB connection when the block ends allowing me to drop the DB.